modules/farm_fd2_school/src/entrypoints/Vue4/App.vue

```diff
@@ -107,6 +107,13 @@
       value="Submit"
       class="label-margin"
       v-on:click="console.log('Submit button clicked.')"
+      v-bind:disabled="
```

Comment:
It is generally preferable to place complex logic into the script as opposed to the template. So here a computed property would be appropriate.